### PR TITLE
get single ID

### PR DESCRIPTION
**Pull request by EvilDrW**
addresses issue #16

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -183,14 +183,20 @@ Fileupload.prototype.handle = function (ctx, next) {
...
 
-        if (this.events.get) {
-            this.events.get.run(ctx, domain, function(err) {
-                if (err) return ctx.done(err);
-                self.get(ctx, next);
-            });
-        } else {
-            this.get(ctx, next);
-        }
+		this.get(ctx, function(err, result) {
```
Comment (NicolasRitouet) [inline index.js]: Nice, you also fixed the issue that the event is called before the actual get request. Was is on purpose?

**Comment by NicolasRitouet** [comment]
Looks good, do you mind adding a test for this feature?

**Comment by EvilDrW** [comment]
I only pretend to be a real software person, but I will add something to
test it.

**Comment by EvilDrW** [comment]
see #40

**Pull request closed without merging**